documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+| [Tabling](#tabling)                                   | N/A                 | Indicate that a proposal will not be implemented                                          |
+
+## Proposal requirements
+
+Project proposals and technical implementation plans should follow the formats
+outlined in [the documentation for those processes](./docs/projects/README.md).
+All proposals should include a summary of the purpose of the proposal and the
+problem(s) it attempts to address.
+
+Every proposal must include the following boilerplate text in the PR description
+or at the top of the GitHub Discussion:
+
+```md
+## Decision-making process
+
+For the purposes of this discussion, we will follow
+[Openverse’s version of the Consent decision-making model](https://wordpress.github.io/openverse/reference/consent_decision_making.html).
+Please note that this process follows formalised steps with specific
+expectations of participants. Before jumping in, please read
+[this document](https://wordpress.github.io/openverse/reference/consent_decision_making.html)
+as well as
+[Openverse’s Code of Conduct](https://github.com/WordPress/openverse/blob/main/CODE_OF_CONDUCT.md).
+Please note that the consent decision-making document linked above also includes
+specific provisions for opting out of a decision discussion you do not wish to
+or cannot participate in.
+
+## Current round
+
+The discussion is currently in the {Question, Reaction, Revision, Objection
+raising, Objection revision} round.
+```
+
+## Round descriptions
+
+Please keep in mind the following general guidelines that apply to all rounds:
+
+- Reactions may be shared at any of the "feedback" rounds (Question and
+  Objection).
+- Proposal authors are not obligated to respond to every question, reaction, or
+  objection that is raised, except for paramount objections. While there is an
+  expectation that the author will address outstanding questions, it isn't
+  necessary to respond to every non-paramount objection before moving on to
+  subsequent rounds or ratifying the proposal. It is, of course, in the best
+  interest of the project and the proposal that as much clarity as possible is
+  achieved.
+- Authors may table a proposal at any time at their discretion. If the proposal
+  is tabled, discussion should not be expected to continue, but may, if there is
+  something important being discussed.
+- Any round with a suggested length of time is subject to
+  [shortcutting as described below](#shortcutting-a-round) or to extension at
+  the discretion of the participants.
+
+### Question round
+
+This round lasts for 3 business days after the discussion starts but can be
+extended if questions reveal significant gaps in the proposal that must be
+understood before the proposal can be revised in the revision step. These
+questions are different from objections in that they’re not in the form of "we
```

Comment:
> These questions are different from objections

Can we introduce the goals of this round before clarifying that questions are not objections? Something like "In this round participants ask questions to clarify their understanding of the proposal and identify gaps." or something to that aim.